package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "node-exercise",
+  "version": "0.0.0",
+  "description": "",
+  "author": "",
+  "license": "ISC",
+  "scripts": {
+    "start": "npm run build && cross-env NODE_ENV=production node build/server.js",
```

Comment:
Hi,
I setup the project with a generator called typescript-express-starter (https://www.npmjs.com/package/typescript-express-starter)
So I could create the project with the command **npm install -g typescript-express-starter**
I did this because I thought it would be good to have an express API and Swagger document for the exercise.